src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -1144,6 +1166,123 @@ export default class SourceFileLinter {
 		}
 	}
 
+	#analyzeNewAndApplySettings(node: ts.NewExpression | ts.CallExpression) {
+		node?.arguments?.filter((arg) => ts.isObjectLiteralExpression(arg))
+			.flatMap((arg) => arg.properties)
+			.forEach((prop) => {
+				if (ts.isPropertyAssignment(prop) &&
+					(ts.isCallExpression(node) ||
+						this.#isPropertyBinding(node, getPropertyNameText(prop.name)))
+				) {
+					if (ts.isObjectLiteralExpression(prop.initializer)) {
+						this.#analyzeControlBindings(prop.initializer, ["type"]);
+					} else {
+						this.#analyzeControlStringBindings(prop);
+					}
+				}
+			});
+	}
+
+	#analyzeControlBindings(node: ts.ObjectLiteralExpression, propNames: string[]) {
```

Comment:
isn't this rather a PropertyBinding that is being analyzed here? AFAIK there's no control binding. Bindings are implemented on ManagedObject level, so if we need a more general name, it shouldn't be control, as also elements can have bindings but are not controls.